packages/settings-library/tests/conftest.py

```diff
@@ -48,7 +48,7 @@ def mocks_folder(project_tests_dir: Path) -> Path:
 @pytest.fixture
 def env_file():
     """Name of env file under tests/mocks folder. Override to change default"""
-    return ".env-compact"
+    return ".env-sample"
```

Comment:
Because out of the box, Pydantic is not compatible with the `compact` format. All the compact and mixed compact formats were removed and this file was renamed to `.env-sample`.

I've also added tests to enforce this behaviour. For now we desired to not support the compact format in the application's environment variables.

Reply:
@GitHK pydantic **does support** what we call compact format. See demo in https://github.com/pcrespov/sample-app-settings.

Reply:
@pcrespov I still have troubles finding where this happens in the sample code you referenced. I might still be missing. Could you please point it out to me?